include/measurement_kit/report/entry.hpp

```diff
@@ -13,7 +13,7 @@ namespace report {
 // A report entry. This is implemented using private inheritance from
 // nlohmann/json such that we are not exposing externally we are using
 // such json library to implement this class.
-class Entry : private nlohmann::json {
+class Entry : public nlohmann::json {
```

Comment:
@hellais if you don't have objections here I'd say "yolo" and "let's expose nlohmann::json" publicly